src/app/store/reducers/mail.reducers.ts

```diff
@@ -30,13 +30,39 @@ export function reducer(state = initialState, action: MailActions): MailState {
     }
 
     case MailActionTypes.MOVE_MAIL_SUCCESS: {
-      // TODO: use immutable version of remove items from array, with modifying the state directly
+      const listOfIDs = action.payload.ids.split(',');
```

Comment:
If we have to split the IDs then why not send the array of IDs instead of comma separated string?